features/support/twopence_init.rb

```diff
@@ -1,50 +1,57 @@
 require 'twopence'
 
-# initialize ssh targets from environment variables.
-
-raise 'Server ip var empty' if ENV['TESTHOST'].nil?
-raise 'client ip var empty' if ENV['CLIENT'].nil?
-raise 'minion ip var empty' if ENV['MINION'].nil?
-raise 'ceos-minion ip var empty' if ENV['CENTOSMINION'].nil?
-raise 'ssh-minion ip var empty' if ENV['SSHMINION'].nil?
+# Initialize SSH targets from environment variables
+# If there is no proxy, use "None" for $PROXY environment variable
+raise 'Server IP address or domain name variable empty' if ENV['TESTHOST'].nil?
+raise 'Proxy IP address or domain name variable empty' if ENV['PROXY'].nil?
+raise 'Client IP address or domain name variable empty' if ENV['CLIENT'].nil?
+raise 'Minion IP address or domain name variable empty' if ENV['MINION'].nil?
+raise 'CentOS minion IP address or domain name variable empty' if ENV['CENTOSMINION'].nil?
+raise 'SSH minion IP address or domain name variable empty' if ENV['SSHMINION'].nil?
 
 $server_ip = ENV['TESTHOST']
+$proxy_ip = ENV['PROXY']
```

Comment:
it would be great if we get rid of Global variables, but ok  for time beeing

Reply:
this one can be not global

Reply:
No, it's used at other places

As you say, we should remove them, but thzen all of them

Reply:
i would rather create a method for ip , so we don introduce new unclean code..

Reply:
Yes, but let's do this kind of refactorings in separate PRs. This is complicated and big enough.